### PR TITLE
Documentation for cooking content on a server

### PR DESCRIPTION
**Pull request by edwoodward**

### CODE REVIEW

File: docs/server-side-cooking.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Instructions for cooking content on the server
+==============================================
+
+ - From cnx-recipies in Github, get URL to raw version of the ruleset needed. Path: cnx-recipes/books/rulesets/output/
```
Comment (reedstrm) [inline docs/server-side-cooking.md]: recipies -> recipes

File: docs/server-side-cooking.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ >https://raw.githubusercontent.com/Connexions/cnx-recipes/master/books/rulesets/output/statistics.css
+
+ - ssh into server
+
+ - wget the ruleset
+ >wget https://raw.githubusercontent.com/Connexions/cnx-recipes/master/books/rulesets/output/statistics.css
+
+ - Pro Tip: Remove ruleset if it already exists before running wget
+
+ - Run this command
```
Comment (reedstrm) [inline docs/server-side-cooking.md]: Run this command to install the ruleset into the book as an attached resource.

Comment (edwoodward) [inline docs/server-side-cooking.md]: Updated

File: docs/server-side-cooking.md (hunk 1 above)
Comment (edwoodward) [inline docs/server-side-cooking.md]: Updated

**Pull request merged**